widgets/linechart/formatter_test.go

```diff
@@ -0,0 +1,117 @@
+package linechart
+
+import (
+	"testing"
+	"time"
+
+	"github.com/kylelemons/godebug/pretty"
+)
+
+func TestFormatters(t *testing.T) {
+	tests := []struct {
+		desc      string
+		value     float64
+		formatter func(float64) string
```

Comment:
(optional) We could reference our type here which will serve as documentation. Test cases are often used when discovering how code works.

```go
formatter ValueFormatter
```